build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+set -e
+
+SQLITE_VERSION="2.7.6"
+POWERSYNC_CORE_VERSION="0.4.0"
+SQLITE_PATH="sql.js"
+
+if [ -d "$SQLITE_PATH" ]; then
+  echo "Deleting existing clone"
+  rm -rf $SQLITE_PATH
+fi
+
+git clone --depth 1 https://github.com/sql-js/sql.js.git $SQLITE_PATH
```

Comment:
I think this should probably clone a fixed revision to ensure we can always run the script and update the core extension even when there are changes from sql.js? Otherwise there's a risk that they break the patches (which we'd have to fix either way, but it'd be nice for this to be reproducible).